set.mm

```diff
@@ -15296,7 +15296,7 @@ only postulated (that is, axiomatic) rule of inference of predicate
   altru $p |- A. x T. $=
     ( wtru tru ax-gen ) BACD $.
 
-  $( For all sets, ` F. ` is not true.  (Contributed by Anthony Hart,
+  $( For all sets, ` -. F. ` is true.  (Contributed by Anthony Hart,
```

Comment:
I would prefer not to change the comment: Both versions are equivalent, but while the new one is more formal, the former one is more intuitive/better understandable.

Reply:
What you say is correct, and I wouldn't change the comment for a more advanced theorem.  Here, it's one of the more elementary results, so I prefer the comment to be more faithful to the formal statement (and of course it's not that faithful, but there has to be a limit before completely paraphrasing the statement).

Also: I haven't completely given up on the idea of a more modular axiomatization (https://github.com/metamath/set.mm/issues/756#issuecomment-877045990) which has paraconsistent logic (that is, without "ex falso quodlibet") as one of its subsystems, where the above statement corresponds to the comment I proposed.

Reply:
OK